packages/documentation/data/menu.ts

```diff
@@ -20,6 +20,7 @@ import {
 	KtPopover,
 	KtTable,
 	KtTag,
+	KtFieldTextArea,
```

Comment:
Why a new page in docs is necessary? We already have documentation for all KtField* components at `packages/documentation/pages/usage/components/form-fields.vue`